### PR TITLE
add document for jubash

### PR DESCRIPTION
**Pull request by kmaehashi**
Docs for https://github.com/jubatus/jubakit/pull/45

### CODE REVIEW

File: doc/source/architecture/shell.rst [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  service = Classifier.run(Config())
+  service.shell()
+
+When :py:func:`shell() <jubakit.base.BaseService.shell>` method is called, an interactive shell to control the Service is started.
+You can interactively call RPC API to the Jubatus server in the shell.
+The jubakit script stops until you exit the shell.
+
+See the next section for the usage of the interactive shell.
+
+Using REPL from Commnand Line
```
Comment (rimms) [inline doc/source/architecture/shell.rst]: typo s/Commnand/Command/

File: doc/source/guide/jubash.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+   Stop when error occurred when running script.
+
+.. option:: -p <prompt>, --prompt <prompt>
+
+   Use specified shell prompt format in interactive mode.  [``[Jubatus:{service}<{cluster}>@{host}:{port}] #``]
+
+.. option:: -v, --verbose
+
+   Enable verbose mode.
+   In verbose mode, the contents of Datum is displayed when making RPC call.
```
Comment (rimms) [inline doc/source/guide/jubash.rst]: the contents of Datum **are** displayed

**Comment by rimms** [comment]
👍 LGTM except for comments above.
``jubash`` is cool!! 😎

**Comment by kmaehashi** [comment]
@rimms Thanks for review, fixed typos.

**Comment by rimms** [comment]
👍

**Pull request merged**